codemeta.json

```diff
@@ -402,7 +450,7 @@
   ],
   "releaseNotes": "https://github.com/ropensci/tic/blob/master/NEWS.md",
   "readme": "https://github.com/ropensci/tic/blob/master/README.md",
-  "fileSize": "350.164KB",
+  "fileSize": "14375.312KB",
```

Comment:
That's a lot of stuff.

Reply:
Indeed - wondering why. Didn't check the number before.

Reply:
Found it - for some reason I still had a `.drake` folder in the repo.